docs/cookbook/recipe_sortable_listing.rst

```diff
@@ -117,16 +117,17 @@ Now we need to define the sort by field to be ``$position``::
     namespace App\Admin;
 
     use Sonata\AdminBundle\Admin\AbstractAdmin;
+    use Sonata\AdminBundle\Datagrid\DatagridInterface;
     use Sonata\AdminBundle\Datagrid\ListMapper;
     use Sonata\AdminBundle\Route\RouteCollection;
 
     final class ClientAdmin extends AbstractAdmin
     {
         protected function configureDefaultSortValues(array &$sortValues): void
         {
-            $sortValues['_page'] = 1;
-            $sortValues['_sort_order'] = 'ASC';
-            $sortValues['_sort_by'] = 'position';
+            $sortValues[DatagridInterface::PAGE] = 1;
+            $sortValues[DatagridInterface::SORT_ORDER] = 'ASC';
```

Comment:
You can use https://github.com/doctrine/collections/blob/1.6.x/lib/Doctrine/Common/Collections/Criteria.php#L18 for `ASC` and `DESC`

Reply:
Not sure about this, the phpdoc of the class says:

```php
/**
 * Criteria for filtering Selectable collections.
 */
```
and the usage inside the `doctrine` organization: https://github.com/search?q=org%3Adoctrine+Criteria%3A%3ADESC&type=Code looks like it's only around the `Criteria` class.